server/bin/www.js

```diff
@@ -113,26 +115,31 @@ for (const hour in config.rabbitmq.sync.hours) {
   } else {
     var syncTime = '00 00 ' + config.rabbitmq.sync.hours[hour] + ' * * 0-6'
   }
-  new CronJob({
-    cronTime: syncTime,
-    onTick: function () {
-      var importer = spawn('node', ['./utils/importer.js'])
-
-      importer.stdout.on('data', (data) => {
-        console.log(`stdout: ${data}`)
-      })
-
-      importer.stderr.on('data', (data) => {
-        console.log(`stderr: ${data}`)
-      })
-
-      importer.on('close', (code) => {
-        console.log(`child process exited with code ${code}`)
-      })
-    },
-    start: true,
-    timeZone: config.rabbitmq.sync.timezone,
-  })
+
+  const importerPath = path.join(__dirname, '..', 'utils', 'importer.js')
+
+  if (fs.existsSync(importerPath)) {
```

Comment:
Hi @Elkrival , I see that you have only added this `if` condition. Does this assure that the issue won't happen?

Reply:
Yes, this checks the existence of the importer.js file, if the file exists then we run the work from the file.